CodeGen/Generators/QuantityJsonFilesParser.cs

```diff
@@ -110,25 +110,17 @@ private static Localization[] GetLocalizationForPrefixUnit(IEnumerable<Localizat
         {
             return localizations.Select(loc =>
             {
-                if (loc.TryGetAbbreviationsForPrefix(prefixInfo.Prefix, out string[] unitAbbreviationsForPrefix))
+                if (loc.TryGetAbbreviationsForPrefix(prefixInfo.Prefix, out var unitAbbreviationsForPrefix))
                 {
-                    return new Localization
-                    {
-                        Culture = loc.Culture,
-                        Abbreviations = unitAbbreviationsForPrefix
-                    };
+                    return new Localization {Culture = loc.Culture, Abbreviations = unitAbbreviationsForPrefix};
                 }
 
                 // No prefix unit abbreviations are specified, so fall back to prepending the default SI prefix to each unit abbreviation:
                 // kilo ("k") + meter ("m") => kilometer ("km")
                 var prefix = prefixInfo.GetPrefixForCultureOrSiPrefix(loc.Culture);
                 unitAbbreviationsForPrefix = loc.Abbreviations.Select(unitAbbreviation => $"{prefix}{unitAbbreviation}").ToArray();
 
-                return new Localization
-                {
-                    Culture = loc.Culture,
-                    Abbreviations = unitAbbreviationsForPrefix
-                };
+                return new Localization {Culture = loc.Culture, Abbreviations = unitAbbreviationsForPrefix};
```

Comment:
You need to revert ffa1d05 with R# cleanup of entire solution :)

The intention was good, but it makes it pretty much impossible to review this PR and it should at any rate be a separate PR. Heck, Chrome browser is so heavily loaded trying to display all the file diffs that I can barely type into this text box 😆 

It's a very good practice to keep PRs small and do changes in small isolated chunks, so we can much more efficiently review them.

Besides, the R# rules are not perfectly configured and results in some changes to formatting I'm not a fan of. We can discuss further if you do a separate PR and maybe we can tweak some rules to make it behave better.

Reply:
I'll review further after the revert, because my browser is really struggling to open up diffs etc.

Reply:
Haha sorry, saw it was set up in the wiki, figured the rules were configured, and my contributions were all that needed cleanup... Will revert and correct!